cmd/soroban-cli/src/toid/mod.rs

```diff
@@ -0,0 +1,69 @@
+/// A barebones implementation of Total Order IDs (TOIDs) from
```

Comment:
*style*
I think if there is only one file, the best practice would be to do:
`cmd/soroban-cli/src/toid.rs`

Reply:
Otherwise this is nice. 👍

Reply:
cool, sure, I was just following `rpc/mod.rs`, and thought this might evolve into its own crate some day

Reply:
Eh. Let's move it if/when that happens. the go toid package has been around a while without risk of expansion.